### DISCUSSION TITLE
archives/51db

### DISCUSSION THREAD

**Discussion by giscus** [SMGoro/issues-json-generator#12] [General]
# archives/51db

因为近期很多腐竹都遭到了DDoS的攻击，本人也不例外。为了帮助广大公益服腐竹，本人总结了一些个人经验，希望能对大家有所帮助。  注：本文重点在于**“如何经济廉价的解决DDoS攻击所带来的影响”**，所以各大厂商高昂的DDoS防御服务不在本文的介绍范围内。 本文集中整理“方法列表”，而具体提及的一些方法的具体使用如果不懂请自行搜索。没有提到的一般都是网上都能搜索到的。 由于云环境多样复杂，本文提

https://blog.smgoro.com/archives/51db

**Comment by Nova-Squ1** [comment]
frp可以使用proxy_protocol_version = v2来反映真实端口吧

**Comment by Nova-Squ1** [comment]
还有现在imc家宽最多防了多少gddos没死？想参考一下
